docs/getting_started/tutorial.rst

```diff
@@ -66,30 +67,32 @@ This example extends the HomePage model to add a body field.
         body = RichTextField(blank=True)
 
         content_panels = Page.content_panels + [
-            FieldPanel('body')
+            FieldPanel('body', classname="full")
         ]
 
 ``body`` is defined as ``RichTextField``, a special Wagtail field. You
 can use any of the `Django core fields <https://docs.djangoproject.com/en/1.8/ref/models/fields/>`__. ``content_panels`` define the
 capabilities and the layout of the editing interface. :doc:`More on creating Page models. <../topics/creating_pages>`
 
-
 Run ``python manage.py makemigrations``, then
 ``python manage.py migrate`` to update the database with your model
 changes. You must run the above commands each time you make changes to
 the model definition.
 
-To reflect the model changes, edit
-``core/templates/core/home_page.html``. Wagtail uses normal Django
-templates to render each page type. It automatically generates a
-suggestion by separating capital letters with underscores (e.g. HomePage
-becomes home\_page.html).
+You can now edit the homepage within the Wagtail admin area (go to Explorer, Homepage, then Edit) to see the new Body field. Enter some text into the body field, and publish the page.
```

Comment:
Good point - have standardised on 'body field' in 34a080b0569732dbf5db73d8335086953db3f88d.